src/js/github-api.js

```diff
@@ -98,23 +105,29 @@ export async function getIssues(token, owner, repo) {
 
     //Create task object from fetched data and populate array to return - return issues
     for (let i = 0; i < data.length; i ++) {
-      const parsedIssue = {
-        type: 'issue',
-        title: String(data[i].title),
-        done: false,
-        dueDate: data[i].updated_at instanceof Date 
-          ? data[i].updated_at 
-          : new Date(data[i].updated_at ),
-        description: String(''),
-        url: String(data[i].url),
-        priority: String('high'),
-        tags: [owner, repo]
-      };
-      arrayIssues.push(parsedIssue);
+      //Get the list of assignees and then parse them into a list of login usernames 
```

Comment:
Since this is the same exact code except pushing to a different list, maybe you can write another function that's only used in this file to reduce code duplication? Just a suggestion.